rotkehlchen/chain/evm/decoding/curve/decoder.py

```diff
@@ -782,14 +790,43 @@ def _maybe_enrich_curve_transfers(self, context: EnricherContext) -> TransferEnr
             return TransferEnrichmentOutput(matched_counterparty=CPT_CURVE)
         return FAILED_ENRICHMENT_OUTPUT
 
+    def decode_gauge_mints(self, context: DecoderContext) -> DecodingOutput:
+        """Decodes minting of CRV happening in L2s via the ChildLiquidityGaugeFactory contract"""
+        if (
+            context.tx_log.topics[0] != MINTED_CRV or
+            not self.base.is_tracked(user := bytes_to_address(context.tx_log.topics[1]))
```

Comment:
```suggestion
            not self.base.is_tracked(user_address := bytes_to_address(context.tx_log.topics[1]))
```
nitpick, but this should probably be `user_address` rather than just `user`.